node/demo_2/controllers/orderController.js

```diff
@@ -4,41 +4,20 @@ const sendResponse = require('../views/view.js');
 
 class OrderController {
     async postOrder(req, res) {
-        const [products, user] = orderModel.parseUrl(req.path);
+        const products = orderModel.parseUrl(req.path);
+        const user  = res.locals.user;
         user.email = user.email.match(/^[^@\s]+@[^@\s]+\.[^@\s]+$/) != null ? user.email : null;
 
-        try {
-            const validateData = orderModel.validateData(user);
-            const isFoundProducts = await orderModel.findProduct(products);
-            const isEnoughAmount = await orderModel.checkAmount(products);
+        const orderId = orderModel.addOrderedItems(products, user.id);
+        const ok = {status: 'ok', data: [], message: 'Order added'};
```

Comment:
`ok` is not very descriptive name for a variable. Maybe `successfulResponse` or something like that.